src/datepicker/engine/flag-days.calendar.spec.ts

```diff
@@ -2,42 +2,85 @@ import { flagDaysCalendar } from './flag-days-calendar';
-              { date: new Date('2019-02-09'), label: '2019-02-09'  }
-            ]}
-        ],
-        weekNumbers: [],
-        weekdays: [],
-        monthTitle: '',
-        yearTitle: ''
-      };
-      const datesDisabled = [
-        new Date('2019-02-07'),
-        new Date('2019-02-09')
-      ];
-      const result = flagDaysCalendar(weekViewModel, {
-        datesDisabled,
-        isDisabled: false,
-        minDate: new Date('2019-01-01'),
-        maxDate: new Date('2019-12-31'),
-        daysDisabled: [],
-        hoveredDate: new Date('2019-02-06'),
-        selectedDate: new Date('2019-02-05'),
-        selectedRange: [],
-        displayMonths: 1,
-        monthIndex: 1,
-        dateCustomClasses: []
-      });
+  it('should flag days as disabled when they are part of the datesDisabled', () => {
+    const weekViewModel = {
+      month: new Date('2019-02-01'),
+      weeks: [
+        {
+          days: [
+            { date: new Date('2019-02-07'), label: '2019-02-07' },
+            { date: new Date('2019-02-08'), label: '2019-02-08' },
+            { date: new Date('2019-02-09'), label: '2019-02-09' }
+          ]
+        }
+      ],
+      weekNumbers: [],
+      weekdays: [],
+      monthTitle: '',
+      yearTitle: ''
+    };
+    const datesDisabled = [
+      new Date('2019-02-07'),
+      new Date('2019-02-09')
+    ];
+    const result = flagDaysCalendar(weekViewModel, {
+      datesDisabled,
+      isDisabled: false,
+      minDate: new Date('2019-01-01'),
+      maxDate: new Date('2019-12-31'),
+      daysDisabled: [],
+      datesEnabled: [],
+      hoveredDate: new Date('2019-02-06'),
+      selectedDate: new Date('2019-02-05'),
+      selectedRange: [],
+      displayMonths: 1,
+      monthIndex: 1,
+      dateCustomClasses: []
+    });
+
+    expect(result.weeks[0].days.find(day => day.label === '2019-02-07').isDisabled).toBe(true);
+    expect(result.weeks[0].days.find(day => day.label === '2019-02-08').isDisabled).toBe(false);
+    expect(result.weeks[0].days.find(day => day.label === '2019-02-09').isDisabled).toBe(true);
+  });
 
-      expect(result.weeks[0].days.find(day => day.label === '2019-02-07').isDisabled).toBe(true);
-      expect(result.weeks[0].days.find(day => day.label === '2019-02-08').isDisabled).toBe(false);
-      expect(result.weeks[0].days.find(day => day.label === '2019-02-09').isDisabled).toBe(true);
+  it('should flag days as disabled when they are not part of the datesEnabled', () => {
+    const weekViewModel = {
+      month: new Date('2020-02-01'),
+      weeks: [
+        {
+          days: [
+            { date: new Date('2020-02-07'), label: '2020-02-07' },
+            { date: new Date('2020-02-08'), label: '2020-02-08' },
+            { date: new Date('2020-02-09'), label: '2020-02-09' }
+          ]
+        }
+      ],
+      weekNumbers: [],
+      weekdays: [],
+      monthTitle: '',
+      yearTitle: ''
+    };
+    const datesEnabled = [
```

Comment:
I don't know where I should add this comment. Let it be here :) What will be if the user set up **datesEnabled** and **datesDisabled** at the same time? What will be if these dates are the same?

Reply:
:thinking: 
I don't expect somebody will use both of this functionalities, that's really strange
If dates are the same, all dates will be disabled at all

Reply:
added a note for both functionalities in doc, about not using both of them at the same time